### PR TITLE
Upgraded deps sqlite3@4.0.1

### PR DESCRIPTION
**Pull request by zce**
refs: #115

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,6 @@
   },
   "optionalDependencies": {
     "mysql": "^2.11.1",
-    "sqlite3": "^4.0.0"
+    "sqlite3": "^4.0.4"
```
Comment (kirrg001) [inline package.json]: Why min version 4.0.4?

Comment (zce) [reply package.json]: 👋 No special reason~

Comment (kirrg001) [reply package.json]: I'd suggest to use the min version which officially supports node 10?

Comment (zce) [reply package.json]: Yes, you are right. I didn't consider it thoughtfully. 😀

Comment (kirrg001) [reply package.json]: 👻 Thanks for your help!!

File: package.json [hunk 2]
```diff
@@ -73,6 +73,6 @@
   },
   "optionalDependencies": {
     "mysql": "^2.11.1",
-    "sqlite3": "^4.0.4"
+    "sqlite3": "4.0.1"
```
Comment (kirrg001) [inline package.json]: What is the reason removing the caret 👻

**Comment by gargol** [comment]
@zce would also add node v10 in Travis configuration in this PR, as that's what this bump seems to be for :wink:

**Comment by kirrg001** [comment]
SQlite3 was already bumped in master. I've added Node v10 to travis.yml.

Thanks for your PR :)

**Pull request closed without merging**